### ISSUE TITLE
Add a Python API

### ISSUE THREAD

**Issue by dfarrell07** [dfarrell07/wcbench#37]
It would be very helpful to support kicking off simple actions via a well-define Python API. For example, starting and stopping ODL.

**Comment by aglenis** [comment]
I would be interested in this.
Do you have some kind of API in mind?

**Comment by dfarrell07** [comment]
> I would be interested in this.

Wonderful! :+1: 

> Do you have some kind of API in mind?

Here are some initial thoughts.

* `install_odl [verbose]`
* `install_cbench [verbose]`
* `start_odl [odl_processors] [verbose]`
* `run_cbench [run length in minutes] [verbose]`
* `stop_odl [verbose]`
* `clean_up [verbose]`

**Comment by dfarrell07** [comment]
> Here are some initial thoughts.

**Comment by dfarrell07** [comment]
> Here are some initial thoughts.

Did those make sense @aglenis? Any other info/help/whatever I can provide?

Thanks!

**Comment by aglenis** [comment]
Makes sense to me.
How exactly should be start implementing it?
My initial though is to wrap every wcbench function with something like this:

http://stackoverflow.com/questions/5826427/can-a-python-script-execute-a-function-inside-a-bash-script

**Comment by dfarrell07** [comment]
> My initial though is to wrap every wcbench function with something like this:

Yeah, that seems reasonable. Only complication may be moving some of the global state in WCBench to params to function calls.
